docs/02_cleos/02_how-to-guides/how-to-submit-a-transaction.md

```diff
@@ -1,18 +1,35 @@
-## Goal
+## Overview
 
-Push a transaction
+This how-to guide provides instructions on how to submit, or push, a transaction using the `cleos` CLI tool.
 
 ## Before you begin
 
 * Install the currently supported version of `cleos`
 
 * Understand the following:
-  * What is a transaction
-  * How to generate a valid transaction JSON
+  * What is a [transaction](https://developers.eos.io/welcome/latest/glossary/index/#transaction).
+  * How to generate a valid transaction JSON.
+    * Consult [cleos push transaction reference](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/push/push-transaction), and pay attention to option `-d` and `-j`.
```

Comment:
The word "reference" should not be included in the link since it's not part of the cleos command.

Reply:
udpated